cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -43,39 +69,51 @@
         </div>
 
     <div class="title-separator"></div>
-    <div class="flex flex-row justify-center" id="genoverse-container">
-        <div id="genoverse"></div>
+
+    <div class="flex justify-center min-w-full max-w-sm">
+        <div class="overflow-x-auto">
```

Comment:
One of these divs (I'm not sure which) should have the id="genoverse-container" attribute.